airgun/views/job_invocation.py

```diff
@@ -28,125 +28,124 @@ def is_displayed(self):
 
 class JobInvocationCreateView(BaseLoggedInView):
     breadcrumb = BreadCrumb()
-    job_category = FilteredDropdown(id='job_invocation_job_category')
-    job_template = FilteredDropdown(locator="//div[contains(@class, 'job_template_selector')]")
-    bookmark = FilteredDropdown(id='targeting_bookmark')
-    search_query = TextInput(name='targeting[search_query]')
-    template_content = ConditionalSwitchableView(reference='job_template')
-
-    @template_content.register('Run Command - SSH Default', default=True)
-    class RunSSHCommandForm(View):
-        command = TextInput(id='command')
 
-    @template_content.register('Power Action - SSH Default')
-    class RestartHostForm(View):
-        action = FilteredDropdown(id='s2id_action')
+    class category_and_template(View):
+        expander = Text(".//button[contains(.,'Category and Template')]")
+        job_category = Select('OUIA-Generated-Select-single-1')
+        job_template = Select('OUIA-Generated-Select-typeahead-1')
 
-    @template_content.register('Puppet Run Once - SSH Default')
-    class RunPuppetForm(View):
-        puppet_options = TextInput(id='puppet_options')
+        def __init__(self, parent, logger=None):
+            """Expand the selected wizard step"""
+            View.__init__(self, parent, logger=logger)
+            self.expander.click()
```

Comment:
This can go to some base class no need to repeat that expander everytime e.g. 
```python 
class WizerdView(View):
    def __init__(self, parent, logger=None):
        """Expand the selected wizard step"""
        View.__init__(self, parent, logger=logger)
        self.expander.click()


class category_and_template(WizerdView):
    expander = Text(".//button[contains(.,'Category and Template')]")
    job_category = Select('OUIA-Generated-Select-single-1')
    job_template = Select('OUIA-Generated-Select-typeahead-1')```